tests/test_separator.py

```diff
@@ -0,0 +1,14 @@
+"""
+ Pylibui test suite.
+
+"""
+
+from pylibui.controls import HorizontalSeparator
+from tests.utils import WindowTestCase
+
+
+class SeparatorTest(WindowTestCase):
```

Comment:
@TiagoDanin please remove this file. I hope this will be the last change request :slightly_smiling_face: